cockatrice/src/game/player/player.cpp

```diff
@@ -3444,9 +3456,16 @@ void Player::actPlay()
 
 void Player::actHide()
 {
-    auto *card = game->getActiveCard();
-    if (card) {
-        card->getZone()->removeCard(card);
+    QList<CardItem *> selectedCards;
+    for (const auto &item : scene()->selectedItems()) {
```

Comment:
Any reason we do this as two separate for loops? Is it for readability?

Reply:
I didn't know if deleting cards within the loop would cause the contents of `selectedItems()` to get modified as I iterated over it. If it doesn't cause any problems, then I'll get rid of the redundant loop.

Reply:
selectedItems returns a list of pointers by value, it does not get modified in the loop